iosxr_iso2vbox.py

```diff
@@ -124,10 +149,10 @@ def main(argv):
     # Handle Input ISO (Local or URI)
     if re.search(':/', args.ISO_FILE):
         # URI Image
-        cmdstring = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
+        cmd_string = 'scp %s@%s .' % (getpass.getuser(), args.ISO_FILE)
```

Comment:
Either form is valid but `.format` is a bit more powerful.

I learned recently that you *should* use %s-style for construction of logging messages because it permits lazy evaluation. For string construction, I tend to prefer .format but I don't have a strong opinion.